Druid/DRCogitStackToRegisterMappingGenerator.class.st

```diff
@@ -171,18 +184,28 @@ DRCogitStackToRegisterMappingGenerator >> visitBytecodeJump: aDRBytecodeJump [
 			 receiver: RBVariableNode selfNode
 			 selector: #genJumpTo:
 			 arguments:
-			 { (self jitCompileTimeExpression: aDRBytecodeJump operand1) })
+			 { (self jitCompileTimeExpression: aDRBytecodeJump operand1) }).
+	
+	"Everything that follows an unconditional jump is potentially dead code"
+	markDeadCode := true
```

Comment:
Was this needed or it's an optimisation? And it is to support CFG with dead code or dead basic blocks? Because an optimised CFG should not have dead code :P

Reply:
This is needed for cogit (It's in DRCogitStackToRegisterMappingGenerator). Cogit has a internal flag saying that the subsequent code is dead. And that flag is set to true when compiling an unconditional jump.
However, if the subsequent instructions are not dead, we need to mark them back as not dead code.